sdks/python/apache_beam/transforms/core.py

```diff
@@ -2332,22 +2332,19 @@ def expand(self, pcoll):
 
     unsafe_reason = trigger.may_lose_data(windowing)
     if unsafe_reason != DataLossReason.NO_POTENTIAL_LOSS:
+      reason_msg = str(unsafe_reason).replace('DataLossReason.', '')
       if pcoll.pipeline.allow_unsafe_triggers:
```

Comment:
Just double checking. Is this (`pcoll.pipeline.allow_unsafe_triggers`) correct? Would not it be something like `pcoll.pipeline.options.allow_unsafe_triggers` ?

Reply:
`options` is marked as deprecated with the message, "References to <pipeline>.options will not be supported."

For that reason, I added a property to access the option:  https://github.com/apache/beam/blob/b0e9f2638cbaca822ee6a58bd8ec0e61db8e799e/sdks/python/apache_beam/pipeline.py#L243